stable/yugaware/values.yaml

```diff
@@ -114,8 +114,14 @@ tls:
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 securityContext:
   enabled: false
-  fsGroup: 1001
-  runAsUser: 1001
+
+  podLevel:
+    fsGroup: 10001
+
+  containerLevel:
+    runAsUser: 10001
+    runAsGroup: 10001
+    runAsNonRoot: true
```

Comment:
This won't give the user much flexibility, and is confusing. Because we will be setting same value for all the containers when we set `containerLevel` and conceptually, podLevel one is supposed to do the same thing i.e. change the securityContext for all the container (though it has extra options like fsGroup). 

How about something like this.
```yaml
securityContext:
  enabled: false
  runAsUser: 10001
  runAsGroup: 10001
  fsGroup: 10001
```

And then, inside every section like `yugaware`, `prometheus`, `nginx`, `postgres` we will have `containerSecurityContext`. This will be blank `{}` for now, i.e. we won't set it explicitly. 

For upgrade scenarios root -> non-root, we will have a comment near `securityContext`, if you are upgrading from root -> non-root setup, make sure you set `postgres.containerSecurityContext.runAsUser` to `999`.

Reply:
Based on offline discussion:
We will have the values I suggested above.
And to make sure we have backward compatibility, we will have following settings by default:
```yaml
postgres:
  containerSecurityContext:
    enabled: true
    runAsUser: 999
```
That way even if someone had `securityContext.enabled=true`, their setup will keep working, because previously their postgres will be running as postgres user (999). 
And we can add a note on `securityContext`: if you would like to have same UID across all the containers then make sure to set `postgres.containerSecurity.enabled=false` while installing the chart.

Reply:
I modified the changes as per the offline discussion.